docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -517,3 +517,7 @@ composer dump-autoload
 ## v4.6.9
 
 No additional steps needed.
+
+## v4.6.10
+
+A command dealing with the duplicated entries caused by [IBX-8562](https://issues.ibexa.co/browse/IBX-8562) will be provided here in the near future.
```

Comment:
A command to deal with the duplicated entries caused by [IBX-8562](https://issues.ibexa.co/browse/IBX-8562) will be available soon.